src/torchcodec/encoders/_video_encoder.py

```diff
@@ -144,6 +159,9 @@ def to_file_like(
                 encoding speed and compression. Valid values depend on the encoder (commonly
                 a string: "fast", "medium", "slow"). Defaults to None
                 (which will use encoder's default).
+            extra_options (dict[str, Any], optional): A dictionary of additional
+                encoder options to pass, e.g. ``{"preset": "slow", "tune": "film"}``.
+                Values will be converted to strings before passing to the encoder.
```

Comment:
Make sure to align the docs (this one still have preset), same with the one below